docs/usage/configuration.md

```diff
@@ -184,4 +184,77 @@ spec:
+
+Alternatively, you can define your own `auditd` rules by using the following configuration:
+```yaml
+apiVersion: rsyslog-relp.extensions.gardener.cloud/v1alpha1
+kind: Auditd
+auditRules: |
+  ## First rule - delete all existing rules
+  -D
+  ## Now define some custom rules
+  -a exit,always -F arch=b64 -S setuid -S setreuid -S setgid -S setregid -F auid>0 -F auid!=-1 -F key=privilege_escalation
+  -a exit,always -F arch=b64 -S execve -S execveat -F euid=0 -F auid>0 -F auid!=-1 -F key=privilege_escalation
+```
+
+To deploy this configuration, it must be embedded in a config map. **Note** that the data key storing this configuration must be named `auditd`.
+An example `ConfigMap` is given below:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: audit-config-v1
+  namespace: garden-foo
+immutable: true
+data:
+  auditd: |
+    apiVersion: rsyslog-relp.extensions.gardener.cloud/v1alpha1
+    kind: Auditd
+    auditRules: |
+      ## First rule - delete all existing rules
+      -D
+      ## Now define some custom rules
+      -a exit,always -F arch=b64 -S setuid -S setreuid -S setgid -S setregid -F auid>0 -F auid!=-1 -F key=privilege_escalation
+      -a exit,always -F arch=b64 -S execve -S execveat -F euid=0 -F auid>0 -F auid!=-1 -F key=privilege_escalation
+```
+
+After creating such a `ConfigMap`, it must be included in the Shoot's `spec.resources` array and then referenced from the `providerConfig.auditConfig.configMapReferenceName` field in the `shoot-rsyslog-relp` extension configuration.
+
+An example configuration is given below:
+
+```yaml
+kind: Shoot
+metadata:
+  name: bar
+  namespace: garden-foo
+...
+spec:
+  extensions:
+  - type: shoot-rsyslog-relp
+    providerConfig:
+      apiVersion: rsyslog-relp.extensions.gardener.cloud/v1alpha1
+      kind: RsyslogRelpConfig
+      target: some.rsyslog-relp.server
+      port: 10250
+      loggingRules:
+      - severity: 7
+      auditConfig:
+        enabled: true
+        configMapReferenceName: audit-config
+  resources:
+    - name: audit-config
+      resourceRef:
+        apiVersion: v1
+        kind: ConfigMap
+        name: audit-config-v1
+```
+
+Finally, by setting `providerConfig.auditConfig.enabled` to `false` in the `shoot-rsyslog-relp` extension configuration, you can completely disable the additional audit configurations performed by the extension.
```

Comment:
IIUC, there are audit rules coming from 3 sources: rules that come with the host OS, rules added by default by the extension and rules that can now be added by the Shoot owners.

When `providerConfig.auditConfig.enabled` won't deploy any rules (extension default ones and Shoot owners ones), right? We can be more explicit here as 
> [...] you can completely disable the additional audit configurations performed by the extension.

it is not super clear what you disable.

Reply:
Just to reply to myself:
- The following deploys the extension default audit rules as of today
```yaml
      auditConfig:
        enabled: true
```

- The following deploys only the rules specified in the ConfigMap
```yaml
      auditConfig:
        enabled: true
        configMapReferenceName: audit-config
```

- The following does not deploy any audit rules
```yaml
      auditConfig:
        enabled: false
        configMapReferenceName: audit-config
```

Reply:
Good point, I could actually list the options like that. I tried simply modifying the text but this is clearer, thanks!